Diagram: add Getting Started tutorial (#6863)
DevExpress/devextreme-documentation: concepts/05 UI Components/Diagram/00 Getting Started with Diagram/00 Getting Started with Diagram.md

#include tutorials-intro-installationnote

> Diagram is a UI component that offers a visual interface to design and edit diagrams.

Reviewer: ```suggestion
Diagram is a UI component that provides a visual interface for creating and editing diagrams.
```

or

```suggestion
Diagram is a UI component for creating and editing diagrams.
```
